src/app.py

```diff
@@ -2128,15 +2128,18 @@ def app_ssowatconf() -> None:
     }
     for domain, apps in portal_domains_apps.items():
         portal_settings = {}
-        portal_settings.update(portal_email_settings)
 
+        # If possible, load the existing file
         portal_settings_path = Path(PORTAL_SETTINGS_DIR) / f"{domain}.json"
         if portal_settings_path.exists():
             this_domain_portal_settings: dict[str, Any] = read_json(
                 str(portal_settings_path)
             )  # type: ignore[assignment]
             portal_settings.update(this_domain_portal_settings)
 
+        # Update with the new settings
+        portal_settings.update(portal_email_settings)
```

Comment:
Eeeh not sure to understand why adding these before or after the other domain settings would matter ?

Reply:
(Cherry-picked the other commit to dev in the meantime)

Reply:
The initial code would prepare the new `portal_settings` (line 2131), then override it with old values on line 2138. We want it the other way around.